pillar/credere_production.sls

```diff
@@ -0,0 +1,34 @@
+network:
+  host_id: ocp24
+  ipv4: 213.52.130.126
+  ipv6: "2a01:7e00:e000:06e6::"
+  networkd:
+    template: linode
+    gateway4: 213.52.130.1
+
+apache:
+  sites:
+    credere:
+      servername: ocp24.open-contracting.org
+
+docker_apps:
+  credere_frontend:
+    target: credere-frontend
+    env:
+      # Vite
+      VITE_API_URL: https://ocp24.open-contracting.org/api # https://credere.open-contracting.org/api
+      VITE_HOST: ocp24.open-contracting.org # credere.open-contracting.org
+  credere_backend:
+    target: credere-backend
+    env:
+      FRONTEND_URL: https://ocp24.open-contracting.org # https://credere.open-contracting.org
+      ENVIRONMENT: production
+      # Timeline
+      APPLICATION_EXPIRATION_DAYS: 7
+      DAYS_TO_CHANGE_TO_LAPSED: 14
+      DAYS_TO_ERASE_BORROWERS_DATA: 7
+      PROGRESS_TO_REMIND_STARTED_APPLICATIONS: 0.7
+      REMINDER_DAYS_BEFORE_EXPIRATION: 3
+      # Email templates
+      LINK_LINK: https://ocp24.open-contracting.org # https://credere.open-contracting.org
+      IMAGES_BASE_URL: https://ocp24.open-contracting.org/images # https://credere.open-contracting.org/images
```

Comment:
I've created the CNAME, so you can replace the ocp24 URLs with the value in the comment (and update the Apache servername).

When deploying, for SSL/TLS, remember to do steps 3 and 4 at https://ocdsdeploy.readthedocs.io/en/latest/develop/update/apache.html#acquire-ssl-certificates